graze/ciff-renderer
# Handle empty text fields correctly

Rendering an empty text field or a text field that contains only invisible characters throws an error.
This is due to the element having a width and height < 1. This PR fixes the bug not rendering these invisible test fields.

### QA
The `.ciff` file included in `/demo` includes an empty text field.

- `php -S localhost:8000`, start the PHP webserver from the base of the project
- browse to http://localhost:8000/demo/demo.php
- ensure the rendered image matches this:

![demo](https://user-images.githubusercontent.com/1314694/32169782-d0f79f96-bd69-11e7-9447-db0880f5491a.png)
